nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/remote/StandardRemoteProcessGroup.java

```diff
@@ -1117,31 +1164,39 @@ public void run() {
                     final String remoteInstanceId = dto.getInstanceId();
                     final boolean isPointingToCluster = flowController.getInstanceId().equals(remoteInstanceId);
                     pointsToCluster.set(isPointingToCluster);
-                } else if (statusCode == UNAUTHORIZED_STATUS_CODE) {
-                    try {
-                        final ClientResponse requestAccountResponse = utils.issueRegistrationRequest(apiUri.toString());
```

Comment:
Hi @mcgilman @markap14 
RPG used to create a NiFi user automatically by sending a request to `/controller/users` which UserResource listens to. But that resource has been removed by #359 and it's not listed on [Support Authorizer API](https://cwiki.apache.org/confluence/display/NIFI/Support+Authorizer+API) resource list.
Does that mean we no longer have to create an user automatically?